stats/src/main/java/io/airlift/stats/DecayTDigest.java

```diff
@@ -151,4 +155,9 @@ public long read()
             }
         };
     }
+
+    public DecayTDigest duplicate()
+    {
+        return new DecayTDigest(TDigest.copyOf(digest), alpha, ticker, landmarkInSeconds);
```

Comment:
`TDigest.copyOf(digest)` or `digest.duplicate()`?

Reply:
There's no `digest.duplicate()`. I went the other way with that class when I wrote it, but I may change it. For Distribution, it's a lot more natural (and almost necessary) due to the synchronization requirements.